components/raftstore/src/store/fsm/store.rs

```diff
@@ -2806,8 +2765,10 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> StoreFsmDelegate<'a, EK, ER
                         delete_ssts.push(sst);
                     }
                 } else {
-                    // If the peer doesn't exist, we need to validate the SST through PD.
-                    validate_ssts.push(sst);
+                    // The write RPC of import sst service have make sure the region do exist at the
+                    // write time, and now the region is not found, sst can be
+                    // deleted because it won't be used by ingest in future.
```

Comment:
How about the upgrade scenario when the SST is uploaded without this change, and then the TiKV is upgraded to this version.

Reply:
Currently I don't know how to solve it 😂 For non-technical solution, it should be OK to forbid importing when upgrading. I'll think about it later.

Reply:
We may need to do some mark in this PR for SSTs uploaded with this change.
And if there's no mark for that file, it means the file is uploaded before this change.